web/crux-ui/src/hooks/use-filters.ts

```diff
@@ -98,7 +98,7 @@ export type EnumFilter<TEnum> = {
 export const enumFilterFor =
   <Item, Enum>(propertiesOf: PropertyValuesOf<Item>): FilterFunction<Item, EnumFilter<Enum>> =>
   (items: Item[], filter: EnumFilter<Enum>) => {
-    if (filter.enum === undefined) {
+    if (filter?.enum === undefined || filter?.enum === null) {
```

Comment:
is there a case where the filter is not declared?
if no then:
```
if (!filter.enum)
```

Reply:
I would set all as a default value then, so we don't have to worry about the filter being null

Reply:
nevermind i got it

Reply:
however:

```
const { enum } = filter
if (!enum)
```